CONTRIBUTING.md

```diff
@@ -1,3 +1,23 @@
-# How to become a contributor and submit your own code
+# How to Contribute
 
-For instructions on how to contribute to the Kubernetes project please refer to the official [contributing guide](https://releases.k8s.io/HEAD/CONTRIBUTING.md)
+We'd love to accept your patches and contributions to this project. There are
```

Comment:
Could you please add something like "before sending a non-trivial PR please create an issue and discuss it there with owners"?